control/control.go

```diff
@@ -445,10 +473,16 @@ func (p *pluginControl) SwapPlugins(in *core.RequestedPlugin, out core.Cataloged
 	return nil
 }
 
+// ExpandWildCards expands wildcards in metric namespace `ns' and returns all matched metrics namespaces based on cataloged metrics
+func (p *pluginControl) ExpandWildCards(ns []string) [][]string {
+	nss, _ := p.metricCatalog.GetMatchedMetrics(ns)
```

Comment:
Thanks, currently, as @jcooklin suggested, the err is returned and if err!= nil the metric from the workflow is used directly.